docs/evidence/mount-latency-20260807/harness/clock-offset.py

```
@@ -0,0 +1,140 @@
+            "%Y-%m-%dT%H:%M:%SZ", time.gmtime()
+        ),
+    }
+    with open(output_path, "a") as raw:
+        raw.write(json.dumps(summary) + "\n")
+    print(json.dumps(summary, indent=2))
+
+
+if __name__ == "__main__":
+    if len(sys.argv) >= 2 and sys.argv[1] == "server":
+        serve(
+            sys.argv[2] if len(sys.argv) > 2 else "0.0.0.0",
```

Ruff (0.16.1)

[error] Possible binding to all interfaces

(S104)